about.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About - Melissa Jimison</title>
+  <meta charset="UTF-8">
+  <link href='https://fonts.googleapis.com/css?family=Shadows+Into+Light' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="css" href="stylesheets/styles.css">
+</head>
+<body>
+
+  <header class="header-about">
+    <nav>
+      <ul>
+        <a class="logo" href="file:///Users/melissajimison/Documents/c5/projects/Static-Site/portfolio.html">Melissa Jimison</a>
```

Comment:
Using relative paths will be more effective - this file path currently breaks for all links and images everywhere except on your machine. Remove the `file:///Users/melissajimison/Documents/c5/projects/Static-Site/` throughout your code and it should work everywhere.